decidim-generators/lib/decidim/generators.rb

```diff
@@ -6,7 +6,7 @@ module Decidim
   module Generators
     def self.edge_git_branch
       if Decidim::Generators.version.match?(/\.dev$/)
-        "develop"
+        "chore/remove-secrets"
```

Comment:
To change once it is approved
```suggestion
        "develop"
```